_source/_posts/2022-08-24-state-of-project-loom.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: blog_post
+title: "What the Heck Is Java Project Loom?"
```

Comment:
```suggestion
title: "What the Heck Is Project Loom for Java?"
```

This title doesn't look quite right, so I tried some others on https://blog-title-analyzer.oktadev.dev/.

- What the Heck Is Project Loom for Java? 69
- What the Heck Is Java's Project Loom? 67

For this reason, I think the former is better.

@havi I thought `Is` should NOT be capitalized, but http://www.conissaunce.com/pages/infoq-title-case.html thinks it should be.